student.rb

```diff
@@ -11,8 +12,16 @@ def initialize(age, classroom, name, parent_permission)
   def play_hooky
     '¯(ツ)/¯'
   end
+
+  def add_classroom(classroom)
+    @classroom = classroom
+  end
```

Comment:
- [ ] You are on the right track to correctly implement the `belongs-to` side. All that is left is to push the student into the `student` array in classroom as long as the student doesn't already exist in the classroom. See hint below 👇 
def add_classroom=(classroom)
      @classroom = classroom
      classroom.student.push(self) unless classroom.student.include?(self)
  end